solutions/landing-zone-v2/landing-zone/namespaces/securitycontrols.md

```diff
@@ -0,0 +1,22 @@
+# Organization Controls - P1
+
+* AC-2(A) - These types of service accounts in this package should be included in the refinement of this control

```

Comment:
AC-2(A) D AC-6(A) - not familiar with Alpha sub controls - usually only numeric like AC-2(1), AC-2(4) in ITSG-33 https://cyber.gc.ca/sites/default/files/cyber/publications/itsg33-ann4a-3-eng.pdf

Reply:
@CraigBaltzer @amcmullin can you guys answer that question from michael